amundsen_application/models/issue_results.py

```diff
@@ -0,0 +1,21 @@
+from amundsen_application.base.base_issue import BaseIssue
+
+
+class IssueResults(dict):
```

Comment:
Do we need to inherit `dict`? (Is `IssueResults` `dict`?)

Reply:
I did this as a workaround to make it serializable without calling a toJSON method. I found adding dict made it work. Very happy to do it differently! Any suggestions?

Reply:
This will be serialized when it's called with `make_response`, right? 
How about using `vars` which returns dictionary based on attributes?
`vars(issue_results)`

Reply:
i get Object of type 'Issue' is not JSON serializable when i try using vars, what I don't quite understand is how the rest of the code works on serialization

Reply:
Is the exception thrown when `vars(issue_results)` is called?

How about:
Option 1 (implement serialize method that returns dictionary):
```
            response = self.client.get_issues(args['key'])
            return make_response(jsonify({'issues': response.serialize()}), HTTPStatus.OK)
```

Or Option 2 (little hacky):
```
            response = self.client.get_issues(args['key'])
            return make_response(jsonify({'issues': response.__dict__}), HTTPStatus.OK)

```

Reply:
yeah it's when the vars() method is called. another option is to inherit from JSONEncoder - I might go that route since that seems the most normal? 

https://stackoverflow.com/questions/10252010/serializing-class-instance-to-json/10252138

Reply:
Yep, sounds good to me.

Reply:
I ended up doing the .serialize() method instead - the way I did it double serialized the 'data issue' class and I didn't want to mess with it further.